scripts/find-lint.sh

```diff
@@ -31,7 +31,7 @@ then args="$args --enable-gc"
 fi
 
 echo "Installing lint search engine..."
-go install github.com/alecthomas/gometalinter/
+gb build github.com/alecthomas/gometalinter/
```

Comment:
apparently this installs gometalinter into `bin` rather than `vendor/bin`; I'm not sure if that is necessarily a bad thing but it certainly surprised me.

Reply:
Given nothing else gets built into `vendor/bin`, i think it makes more sense like that?

Reply:
I think you're probably right, but a couple of thoughts:

1: It's a bit of a gotcha as you can end up with different versions in `vendor/bin` and `bin`, which contributed to my hell yesterday until I figured out what was going on.

2. It's a shame that `bin` fills up with all sorts of crap rather than just our own build products. If I want to build dendrite and tar it up to transfer to another machine, then I have to pick and choose what I want from `bin` rather than just taking the whole dir. It would be nice to separate our own build products from the lint scripts. I'm not sure I have a good way of doing that, but I feel like perhaps this is a backwards step.